moped-editor/src/views/projects/projectsListView/ProjectsListViewTable.js

```diff
@@ -478,6 +545,13 @@ const ProjectsListViewTable = ({ title, query, searchTerm, referenceData }) => {
                         />
                       );
                     },
+                    Header: props => (
+                      <MTableHeader
+                        {...props}
+                        onOrderChange={handleTableHeaderClick}
+                        orderDirection={sort.order}
```

Comment:
turns out all I needed to do to inform arrow direction was pass along this sort order, and material table does the column id checking etc. 

It does change all of the arrows' directions, but that is the same behavior as the normal sort icon in default material table.